src/assets/stylesheets/_navigation.scss

```diff
@@ -70,23 +70,25 @@
   }
 
   > li {
+    transition: background-color 0.15s ease-in-out;
+
     &:hover > a,
     &:focus > a {
       background: transparent;
     }
 
-    &.open > {
-      a,
-      .menu-toggle {
-        background: $nav-hover !important;
+    &.open {
+      background: $nav-hover !important;
+      > a,
+      > .menu-toggle {
         color: #fff !important;
       }
     }
 
     &.active {
+      background: $primary !important;
       > a,
       > .menu-toggle {
-        background: $primary !important;
```

Comment:
* [ ] The blog tag nav https://deploy-preview-472--scrivito-example.netlify.app/blog and search results https://deploy-preview-472--scrivito-example.netlify.app/search-results?q=lorem look broken
<img width="597" alt="image" src="https://user-images.githubusercontent.com/986170/165051761-e1df3ec4-554f-4736-9f9e-c4439c56e79a.png">